app/preprints/-components/submit/metadata/template.hbs

```diff
@@ -1,3 +1,47 @@
 <div local-class='preprint-input-container  {{if (is-mobile) 'mobile'}}'>
-    {{t 'preprints.submit.step-two.title'}}
+    <h3 local-class='title'
+        data-test-title
+    >
+        {{t 'preprints.submit.step-two.title'}}
+    </h3>
+    <div local-class='form-container'>
+        <FormControls
+            @changeset={{this.metadataFormChangeset}}
+            data-test-form
+            as |form|
+        >
+            <form.text
+                data-test-publication-do-input
+                @label={{t 'preprints.submit.step-two.publication-doi-input'}}
+                @ariaLabel={{t 'preprints.submit.step-two.publication-doi-input'}}
+                @valuePath={{'publicationDoi'}}
+                {{on 'change' this.validate}}
+                {{on 'focusout' this.validate}}
+                local-class='input-container'
+                @placeholder={{t 'preprints.submit.step-two.publication-doi-input'}}
+            />
+
+            <form.text
+                data-test-publication-year-input
+                @label={{t 'preprints.submit.step-two.publication-year-input'}}
+                @ariaLabel={{t 'preprints.submit.step-two.publication-year-input'}}
+                @valuePath={{'publicationYear'}}
+                {{on 'change' this.validate}}
+                {{on 'focusout' this.validate}}
+                local-class='input-container'
+                @placeholder={{t 'preprints.submit.step-two.publication-year-input'}}
+            />
+
+            <form.text
```

Comment:
I know this is in the mocks, but I don't think this is needed. The Preprint model doesn't have this `publication_citation` field on it, so I think this was just an extraneous field that snuck its way into the mocks and we just hadn't caught it.